Wikipedia.xcodeproj/project.pbxproj

```diff
@@ -6373,6 +6373,8 @@
 				RUN_CLANG_STATIC_ANALYZER = NO;
 				SWIFT_OBJC_BRIDGING_HEADER = "Wikipedia/Code/Wikipedia-Bridging-Header.h";
 				WMF_HOCKEYAPP_IDENTIFIER = 2295c3698bbd0b050f257772dd2bdbb2;
```

Comment:
Hmm… thats our beta one. The production one really needs to be more secure, but really this shouldn't be there either.